examples/hallmarks_of_cancer.py

```diff
@@ -1,175 +1,189 @@
 
         return [
             datasets.SplitGenerator(
                 name=datasets.Split.TRAIN,
-                gen_kwargs={
-                    "filepath": data_dir,
-                    "split": "train",
-                }
+                gen_kwargs={"filepath": data_dir, "split": "train",},
             )
         ]
 
     def _generate_examples(self, filepath, split):
 
-        path_name = list(filepath.values())[0]+'/*'
-        texts = glob.glob(path_name + '/text/*')
-        labels = glob.glob(path_name + '/labels/')
+        path_name = list(filepath.values())[0] + "/*"
+        texts = glob.glob(path_name + "/text/*")
+        labels = glob.glob(path_name + "/labels/")
         uid = 1
 
         for idx, tf_name in enumerate(texts):
             filenname = os.path.basename(tf_name)
-            with open(tf_name, encoding='utf-8') as f:
+            with open(tf_name, encoding="utf-8") as f:
                 lines = f.readlines()
                 text_body = "".join([j.strip() for j in lines])
 
-            label_file_name = labels[0] + '/' + filenname
-            with open(label_file_name, encoding='utf-8') as f:
+            label_file_name = labels[0] + "/" + filenname
+            with open(label_file_name, encoding="utf-8") as f:
                 lines = f.readlines()
                 label_body = "".join([j.strip() for j in lines])
-                label_body = [i.strip() for i in label_body.split('<')]
-                label_body = sum([k.split('AND') for k in label_body if len(k)>1], [])
-                label_body = [i.split('--')[0].strip() for i in label_body]
+                label_body = [i.strip() for i in label_body.split("<")]
+                label_body = sum([k.split("AND") for k in label_body if len(k) > 1], [])
+                label_body = [i.split("--")[0].strip() for i in label_body]
 
-            if self.config.name == "source":        
+            if self.config.schema == "source":
                 yield idx, {
-                                'document_id': filenname.split('.')[0],
-                                'text': text_body,
-                                'label': label_body
-                            
-                            }
-            elif self.config.name == "bigbio":
+                    "document_id": filenname.split(".")[0],
+                    "text": text_body,
+                    "label": label_body,
+                }
+            elif self.config.schema == "bigbio_text":
 
                 yield idx, {
-                                'id':uid,
-                                'document_id': filenname.split('.')[0],
-                                'text': text_body,
-                                'label': label_body
+                    "id": uid,
+                    "document_id": filenname.split(".")[0],
+                    "text": text_body,
+                    "label": label_body,
                 }
 
-                uid+=1
-
+                uid += 1
```

Comment:
I think we'll want to update the `task_schemas.md` text around `id` fields being unique within each split (around line 45 in "Schema Notes")